resources/wikia/libraries/jquery/loadLibrary/jquery.wikia.loadLibrary.js

```diff
@@ -122,10 +122,14 @@
 	 * @returns {jQuery} Returns a jQuery promise
 	 * @see https://developers.facebook.com/docs/javascript/quickstart/v2.2
 	 */
-	$.loadFacebookAPI = function () {
+	$.loadFacebookAPI = function (callback) {
 		// create our own deferred object to resolve after FB.init finishes
 		var $deferred = $.Deferred();
 
+		if (typeof(callback) === 'function') {
```

Comment:
typeof is an operator, not a function, so I don't think that brackets are necessary here.